src/almanac/introduction.md

```diff
@@ -1,3 +1,72 @@
 # Welcome
 
 Welcome to the Software Gardening Almanac, an open-source handbook of applied guidance and tools for sustainable software development and maintenance.
+
+## Inspiration
+
+```{figure} assets/software-lifecycle.png
```

Comment:
This png is not rendering for me on GitHub markdown. Maybe this is ok since the reader will not view on GitHub?

Reply:
Thanks, yes I expect this to have challenges rendering in GitHub. My hope is to improve PR review capacity on the published content within #26 .